src/pages/index.js

```diff
@@ -13,53 +15,181 @@ import {
   addNewCardButton,
   profileNameInput,
   profileDescriptionInput,
+  profileAvatarEditButton,
+  previewImage,
 } from "../utils/constant.js";
 
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      const card = createCard(cardData);
-      section.addItem(card);
-    },
-  },
-  ".gallery__cards"
+const userInfo = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__avatar"
 );
-section.renderItems();
 
-const addCardForm = document.forms["card-form"];
-const addCardFormValidator = new FormValidator(options, addCardForm);
-addCardFormValidator.enableValidation();
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c2e4b34f-c221-48a4-b144-59557b6d0818",
+    "Content-Type": "application/json",
+  },
+});
+
+let section;
 
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, initialCards]) => {
+    userInfo.setUserInfo({ name: userData.name, about: userData.about });
+    userInfo.setAvatar(userData.avatar);
+    section = new Section(
+      {
+        items: initialCards,
+        renderer: renderCard,
+      },
+      ".gallery__cards"
+    );
+    section.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+function createCard(cardData) {
+  const card = new Card(cardData, "#card-template", {
+    handleImageClick: () => imagePopup.open(cardData.name, cardData.link),
+    handleCardDelete,
+    handleCardLike: handleCardLike,
+  });
+  return card.getView();
+}
+
+function renderCard(cardData) {
+  const cardElement = createCard(cardData);
+  section.addItem(cardElement);
+}
+
+// form validations
+const changeProfileAvatarForm = document.forms["avatar-form"];
 const editProfileForm = document.forms["profile-form"];
+const addCardForm = document.forms["card-form"];
+
+const changeProfileAvatarFormValidator = new FormValidator(
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```